qulice-pmd/src/test/resources/com/qulice/pmd/StaticPublicVoidMainMethod.java

```diff
@@ -0,0 +1,12 @@
+package foo;
+
+public class StaticPublicVoidMainMethod {
+
+    public static void main(final String... args) {
```

Comment:
@vkuchyn Please add another example of accepted main method with this signature `public static void main(String[] args)`